compiler/build.gradle

```diff
@@ -54,47 +54,68 @@ def addLibraryIfNotLinked = { libName, argList ->
 }
 
 model {
-  toolChains {
-    gcc(Gcc) {
-      target("x86_64") {
-        cppCompiler.withArguments { args ->
-          args << "-m64"
-        }
-        linker.withArguments { args ->
-          args << "-m64"
-        }
+  if (System.getProperty('use.vc') != null) {
```

Comment:
I have updated the PR so that it will build 64-bit codegen if osdetector returns x86_64.

I still need to have to switch between VC and gcc/clang in ``toolChains``. If I add everything and happen to have both VC and gcc installed, it will select VC and there is no way to use gcc.

Also removed definitions of built-in targets for gcc and clang.